src/geojson/geojson_intf.ml

```diff
@@ -241,6 +241,10 @@ module type S = sig
   (** [v geojson bbox] combines geojson and bbox to return a GeoJSON object (a
       type {!t}) *)
 
+  val foreign_members : t -> (string * json) list option
```

Comment:
The compilation warning is referring to this. You have told the compiler/type-checker that there should be a function called `foreign_members` which takes a `t` and could produce a value for the foreign members. The in the `.ml` file you haven't provided an implementation for that function.

Reply:
How to do it now...not getting @patricoferris